USB: Product ID assigned to wpanusb 802.15.4 over USB

finikorg opened the pull request:
Assign Product ID and fix using old VID / PID pairs.

finikorg committed:
wpanusb: Assign USB Product ID to 802.15.4 over USB

Assign special Product ID (0x101) to wpanusb protocol exporting
IEEE802.15.4 over USB.

Signed-off-by: Andrei Emeltchenko <andrei.emeltchenko@intel.com>

finikorg committed:
wpanusb: Remove old hardcoded VID / PID

Use values defined in USB Device configuration

Signed-off-by: Andrei Emeltchenko <andrei.emeltchenko@intel.com>

finikorg committed:
wpanusb: Use DEBUG syslog level

Remove magic number

Signed-off-by: Andrei Emeltchenko <andrei.emeltchenko@intel.com>

pfl commented on samples/net/wpanusb/prj.conf:
```diff
@@ -1,8 +1,9 @@
 CONFIG_ARC_INIT=n
 CONFIG_USB=y
 CONFIG_USB_DEVICE_STACK=y
-CONFIG_SERIAL=y
```
Why is CONFIG_SERIAL removed in a patch that only changed PID?

finikorg replied:
That `CONFIG_SERIAL` gave warning that it was already set by other options, I included it in the same patch since the change is small, I can, of course, move this change to separate patch.

pfl replied:
It should rather have its own patch (or be a documented part of another); as it is in here it looks fishy.

finikorg replied:
done

pfl replied:
Thanks, looks logical now.

finikorg committed:
Bluetooth: hci_usb: Use USB Device defined VID / PID

Remove hardcoded INTEL_VENDOR_ID and PRODUCT_ID and use configuration
values.

Signed-off-by: Andrei Emeltchenko <andrei.emeltchenko@intel.com>

finikorg committed:
usb: dfu: Use USB Device defined VID / PID

Remove hardcoded DFU_VENDOR_ID and DFU_PRODUCT_ID and use
configuration values.

Signed-off-by: Andrei Emeltchenko <andrei.emeltchenko@intel.com>

finikorg committed:
wpanusb: Remove unneeded configuration option

Remove unneeded configuration option reported during build.

Signed-off-by: Andrei Emeltchenko <andrei.emeltchenko@intel.com>
